scripts/run-cni-release-tests.sh

```diff
@@ -36,7 +36,12 @@ function run_integration_test() {
   cd $INTEGRATION_TEST_DIR/cni && CGO_ENABLED=0 ginkgo $EXTRA_GINKGO_FLAGS -v -timeout 60m --no-color --fail-on-pending -- --cluster-kubeconfig="$KUBE_CONFIG_PATH" --cluster-name="$CLUSTER_NAME" --aws-region="$REGION" --aws-vpc-id="$VPC_ID" --ng-name-label-key="$NG_LABEL_KEY" --ng-name-label-val="$NG_LABEL_VAL" --test-image-registry=$TEST_IMAGE_REGISTRY || TEST_RESULT=fail
   echo "cni test took $((SECONDS - START)) seconds."
 
-  : "${CNI_METRICS_HELPER:=602401143452.dkr.ecr.us-west-2.amazonaws.com/cni-metrics-helper:v1.13.4}"
+  if [[ ! -z $PROD_IMAGE_REGISTRY ]]; then
```

Comment:
Shouldn't this use the same `TEST_IMAGE_REGISTRY` as above?

Reply:
They are two different registries. We don't share cni images in the `TEST_IMAGE_REGISTRY`

Reply:
Ah, I see now, thanks for clarifying